welkin/models/calendar.py

```diff
@@ -117,3 +127,67 @@ def get(
         return super().get(
             f"{self._client.instance}/calendar/{route}", params=params, *args, **kwargs
         )
+
+
+class WorkHours(Resource):
+    @property
+    def id(self, value=None):
+        if not hasattr(self, "details"):
+            self.details = [{}]
+
+        self.details[0]["workHoursId"] = value
+
+    def create(self, repeating=True):
+        self.startDateTime = datetime.now(tz=timezone.utc)
+        if repeating:
+            self.endDateTime = self.startDateTime.replace(
+                year=self.startDateTime.year + 50
+            )
```

Comment:
We like to be as agnostic as possible with the wrapper and leave specific implementation up to the user in their own project so you can delete these here